framework/src/loops/ComputeUserObjectsThread.C

```diff
@@ -185,6 +189,32 @@ ComputeUserObjectsThread::onInternalSide(const Elem * elem, unsigned int side)
       uo->execute();
 }
 
+void
+ComputeUserObjectsThread::onInterface(const Elem * elem, unsigned int side, BoundaryID bnd_id)
+{
+  // Pointer to the neighbor we are currently working on.
+  const Elem * neighbor = elem->neighbor_ptr(side);
+
+  std::vector<UserObject *> userobjs;
+  queryBoundary(Interfaces::InterfaceUserObject, bnd_id, userobjs);
+  if (_interface_user_objects.size() == 0)
+    return;
+
+  _fe_problem.prepareFace(elem, _tid);
```

Comment:
@friedmud I saw your reply for #12892 adn there was discussion here #12447 about the condition to apply
what is the right if statement here?

```
 if (!((neighbor->active() && (neighbor->level() == elem->level()) ||
        (neighbor->level() < elem->level())))
```

or something else?

Reply:
We had a discussion but did not conclude this. But I guess a temporary solution is:
```
if (neighbor->active())
  // do your stuff here
else
  mooseError("Interface objects do not support sides with hanging nodes due to mesh adaptivity");
```
@friedmud might want to make a comment here.